lua/autorun/client/b-draw_lib.lua

```diff
@@ -145,24 +142,38 @@ end
 ---
 -- Draws a SteamAvatar while caching it before
 -- @param string id64 the steamid64
--- @param number size
+-- @param string size the avatar's size, this can be <code>small</code>, <code>medium</code> or <code>large</code>
+-- @param string url the url to the WebImage
 -- @param number x
 -- @param number y
 -- @param number width
 -- @param number height
 -- @param Color color
--- @param Angle ang
--- @param boolean corner
-function draw.SteamAvatar(id64, size, x, y, width, height, color, ang, corner)
-	draw.WebImage(fetchAvatarAsset(id64, size), x, y, width, height, color, ang, corner)
+-- @param Angle angle
+-- @param boolean cornerorigin if it is set to <code>true</code>, the WebImage will be centered based on the x- and y-coordinate
+function draw.SteamAvatar(id64, size, x, y, width, height, color, angle, cornerorigin)
```

Comment:
This is close to the code of the `draw.WebImage` function and only differs in their material, maybe create a shared draw function and pass the material to be used via a parameter?